xExtension-ReadingTime/static/readingtime.js

```diff
@@ -23,15 +23,15 @@
          reading_time.words_count = reading_time.flux_words_count(flux_list[i]); // count the words
          reading_time.reading_time = reading_time.calc_read_time(reading_time.words_count, 300); // change this number (in words) to your prefered reading speed
 
+
```

Comment:
Stray newline?

Reply:
I don't understand, sorry ^^

Reply:
I mean there was one newline and now there are two for no apparent reason. :-)

Reply:
Oh, yeah, probably a comment line that I did not removed completely ^^
(does it matters ? ^^)

Reply:
It just makes the code more clear ;)